homeassistant/components/axis/device.py

```diff
@@ -313,9 +313,9 @@ async def get_device(hass, host, port, username, password):
         LOGGER.warning("Connected to device at %s but not registered", host)
         raise AuthenticationRequired from err
 
-    except (asyncio.TimeoutError, axis.RequestError) as err:
+    except (asyncio.TimeoutError, axis.RequestError):
         LOGGER.error("Error connecting to the Axis device at %s", host)
-        raise CannotConnect from err
+        raise CannotConnect
```

Comment:
Pylint will probably complain here about this change.

Reply:
I didn't know it was expected, the error is empty so was no value in it. Lets see :)

Reply:
I will keep it for pylints sake